### PR TITLE
logout button

### PR DESCRIPTION
**Pull request by HassenH1**

### CODE REVIEW

File: app/src/pages/GuestQuestions/GuestQuestions.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from 'react'
```
Comment (arshiamasih) [inline app/src/pages/GuestQuestions/GuestQuestions.tsx:1]: looks like your demo files for formik got in here - can you delete them out of this branch?

File: app/src/App.tsx [hunk 2]
```diff
@@ -51,92 +54,101 @@ export const LoginView = () => {
...
-                        </React.Fragment>
-                    </BrowserRouter>
-                </HostHomeDataProvider>
-            ) : (
-                <LoginView />
-            )}
-        </React.Fragment>
-    )
+  const logoutClick = () => {
+    logout()
```
[the remark this answers is not in the document]
Comment (HassenH1) [reply app/src/App.tsx:60]: Yeah I pulled before I pushed my code up.

File: app/src/pages/GuestQuestions/GuestQuestions.tsx (hunk 1 above)
Comment (HassenH1) [reply app/src/pages/GuestQuestions/GuestQuestions.tsx:1]: Well do when I get home

File: app/src/App.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (arshiamasih) [reply app/src/App.tsx:60]: ooo ok can you check (when you can) if you have prettier installed and/or if the formatting has been applied to app? there should be a .prettierrc.js file in there as well. thanks!

File: app/src/pages/GuestQuestions/GuestQuestions.tsx (hunk 1 above)
Comment (HassenH1) [reply app/src/pages/GuestQuestions/GuestQuestions.tsx:1]: so i checked everything
Screenshot (39)

Comment (HassenH1) [reply app/src/pages/GuestQuestions/GuestQuestions.tsx:1]: i also have the prettierrc file as well

**Pull request merged**